src/VisualStudio/Core/Test.Next/Services/SolutionServiceTests.cs

```diff
@@ -796,6 +795,40 @@ public async Task TestPartialProjectSync_ReferenceToNonExistentProject()
             var project1Checksum = await solution.State.GetChecksumAsync(project1.Id, CancellationToken.None);
         }
 
+        [Fact]
+        public async Task TestRemoteWorkspaceStreamingApi1()
+        {
+            var code = @"class Test { void Method() { } }";
+
+            // create base solution
+            using var workspace = TestWorkspace.CreateCSharp(code);
+            using var remoteWorkspace = CreateRemoteWorkspace();
+
+            // create solution service
+            var solution1 = workspace.CurrentSolution;
```

Comment:
would be interesting if we could call through the `RemoteHostClient` code as well to ensure we're keeping everything on the client side around appropriately.
 https://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/Remote/RemoteHostClient.cs,107

Reply:
indeedly.